### PR TITLE
feat(docs): update docs 

### PR DESCRIPTION
**Pull request by Teapane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,3 +85,16 @@ npm version patch -m "Bumped to %s"
 npm version minor -m "Bumped to %s"
 npm version major -m "Bumped to %s"
```
Comment (colinrymer) [inline README.md]: This stuff needs to be removed.

Comment (Teapane) [inline README.md]: Totally.

Comment (aribouius) [inline README.md]: We'll need to update the README template in our generator as well.

Comment (Teapane) [inline README.md]: @aribouius, sounds good. I'll do that real quick.

**Comment by colinrymer** [comment]
can you squash these into just a docs commit?

**Comment by colinrymer** [comment]
:+1:

**Pull request merged**